layouts/authors/list.html

```diff
@@ -33,13 +33,12 @@ <h1 class="mt-2 text-base text-gray-400 md:mt-4">{{ .Params.designation }}</h1>
             </div>
         </div>
         <div class="flex flex-col md:pl-5 w-full md:w-1/2">
-            <h1 class="text-xl font-bold text-sky-900 my-2 md:my-4 ">Bio</h1>
```

Comment:
Now that some of the bios are longer, I didn't want the text to hang too much below the image. The images are at the full height, so removing this label helped some.